src/exchange/group/delegation/group-delegation.controller.js

```diff
@@ -146,5 +156,40 @@ angular.module('Module.exchange.controllers').controller(
           this.services.navigation.resetAction();
         });
     }
+
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
```

Comment:
You will ended up with an ESLint `semi` error.
```suggestion
      const name = _.last(this.currentAccount.split('@'));
```